<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-24T18:14:07 IST" name="Default suite" finished-at="2024-10-24T18:14:28 IST" duration-ms="20310">
    <groups>
      <group name="Smoke">
        <method signature="TestCase_1.errorMsg_Positive()[pri:1, instance:testCases.TestCase_1@44c03695]" name="errorMsg_Positive" class="testCases.TestCase_1"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2024-10-24T18:14:07 IST" name="Default test" finished-at="2024-10-24T18:14:28 IST" duration-ms="20310">
      <class name="testCases.TestCase_1">
        <test-method is-config="true" signature="configureAppium()[pri:0, instance:testCases.TestCase_1@44c03695]" started-at="2024-10-24T18:14:07 IST" name="configureAppium" finished-at="2024-10-24T18:14:28 IST" duration-ms="20229" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\User\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.1
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\User\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\User\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 0.725s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at doListen (node:net:2116:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\User\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.1
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\User\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\User\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 0.725s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at doListen (node:net:2116:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

at testUtils.BaseTest.configureAppium(BaseTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\User\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.1
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\User\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\User\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 0.725s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at doListen (node:net:2116:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)
at testUtils.BaseTest.startAppiumService(BaseTest.java:59)
at testUtils.BaseTest.configureAppium(BaseTest.java:37)
... 26 more
Caused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionTimeout: ConnectionTimeout
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:72)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)
... 28 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:547)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:602)
at java.base/java.net.Socket.connect(Socket.java:633)
at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:178)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:534)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:639)
at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:282)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:387)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:409)
at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1309)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1242)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1128)
at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1057)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.connectToUrl(AppiumServerAvailabilityChecker.java:79)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:61)
... 30 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configureAppium -->
        <test-method signature="errorMsg_Positive()[pri:1, instance:testCases.TestCase_1@44c03695]" started-at="2024-10-24T18:14:28 IST" name="errorMsg_Positive" finished-at="2024-10-24T18:14:28 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\User\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.1
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\User\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\User\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 0.725s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at doListen (node:net:2116:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\User\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.1
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\User\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\User\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 0.725s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at doListen (node:net:2116:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

at testUtils.BaseTest.configureAppium(BaseTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:178)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://127.0.0.1:4723/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.
Node.js executable path: C:\Program Files\nodejs\node.exe
Arguments: [C:\Users\User\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1]
Server log: [38;5;112m[Appium][0m Welcome to Appium v2.11.1
[38;5;112m[Appium][0m Non-default server args:
[38;5;112m[Appium][0m { address: [32m'127.0.0.1'[39m }
[38;5;112m[Appium][0m The autodetected Appium home path: C:\Users\User\.appium
[38;5;112m[Appium][0m Attempting to load driver uiautomator2...
[38;5;112m[Appium][0m Requiring driver at C:\Users\User\.appium\node_modules\appium-uiautomator2-driver\build\index.js
[38;5;112m[Appium][0m AndroidUiautomator2Driver has been successfully loaded in 0.725s
[38;5;86m[HTTP][0m Could not start REST http interface listener. The requested port may already be in use. Please make sure there is no other instance of this server running already.
[38;5;112m[Appium][0m Could not configure Appium server. It's possible that a driver or plugin tried to update the server and failed. Original error: listen EADDRINUSE: address already in use 127.0.0.1:4723
[38;5;112m[Appium][0m Error: listen EADDRINUSE: address already in use 127.0.0.1:4723
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at doListen (node:net:2116:7)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)
at testUtils.BaseTest.startAppiumService(BaseTest.java:59)
at testUtils.BaseTest.configureAppium(BaseTest.java:37)
... 26 more
Caused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionTimeout: ConnectionTimeout
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:72)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)
... 28 more
Caused by: java.net.ConnectException: Connection refused: no further information
at java.base/sun.nio.ch.Net.pollConnect(Native Method)
at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
at java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:547)
at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:602)
at java.base/java.net.Socket.connect(Socket.java:633)
at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:178)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:534)
at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:639)
at java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:282)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:387)
at java.base/sun.net.www.http.HttpClient.New(HttpClient.java:409)
at java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1309)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1242)
at java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1128)
at java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1057)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.connectToUrl(AppiumServerAvailabilityChecker.java:79)
at io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:61)
... 30 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- errorMsg_Positive -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TestCase_1@44c03695]" started-at="2024-10-24T18:14:28 IST" name="tearDown" finished-at="2024-10-24T18:14:28 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TestCase_1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
